ci/firebase_testlab.py

```diff
@@ -11,7 +11,7 @@
 import subprocess
 import sys
 
-BUCKET = 'gs://flutter_firebase_testlab'
+BUCKET = 'gs://flutter_firebase_testlab_staging'
```

Comment:
nit: should this just be an env variable coming from LUCI?

Reply:
SGTM. Let's put the project as an env as well. /cc @ricardoamador

Reply:
On it.